### PR TITLE
Add pages with placeholder content

### PR DESCRIPTION
**Pull request by sophiebrown01**
This PR adds Trending and Phrase pages. The Trending page shows an overview of trending pages, phrases, tags and user groups, whilst the Phrase page drills down into details for a specific phrase.

The majority of the data which these pages currently show is placeholder content, especially for the Phrase page. This will be swapped out for actual data in later commits.

Trello: https://trello.com/c/SGiWePCV, https://trello.com/c/yc5eCMQ4

### CODE REVIEW

File: app/controllers/pages_visited_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class PagesVisitedController < ApplicationController
+  def show
+    @phrase = Phrase.find(params[:id])
+    @unique_visitors_by_page = Page.find_by_sql('select pages.*, count(distinct(page_visits.visit_id)) as unique_visitors from pages join page_visits on page_visits.page_id = pages.id group by pages.id order by unique_visitors desc')
+    @@unique_visitors_by_page = []
```
Comment (oscarwyatt) [inline app/controllers/pages_visited_controller.rb]: Is this class variable meant to be here?

File: app/controllers/phrase_usage_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class PhraseUsageController < ApplicationController
+  def show
+    @phrase = Phrase.find(params[:id])
+    @survey_answers_containing_phrase = SurveyAnswer.find_by_sql(["select * from survey_answers sa join questions q on q.id = sa.question_id join survey_phrases sp on sp.survey_answer_id = sa.id join phrases p on p.id = sp.phrase_id where p.id = ? and q.question_number = 3 and sa.answer not like '-' limit 10", "#{params[:id]}"])
+    # @survey_answers_containing_phrase = Phrase.find_by_sql('select phrase.* from survey_answers sa join questions q on q.id = sa.question_id join survey_phrases sp on sp.survey_answer_id = sa.id join phrases p on p.id = sp.phrase_id where q.question_number = 3 and sa.answer not like \'-\' limit 10')
```
Comment (oscarwyatt) [inline app/controllers/phrase_usage_controller.rb]: If this isn't required/doesn't work I'd remove it, keep a local copy if you want to hang onto it

**Review by oscarwyatt — CHANGES_REQUESTED** [review]
A couple of minor things, otherwise looks good

File: app/controllers/pages_visited_controller.rb (hunk 1 above)
Comment (sophiebrown01) [reply app/controllers/pages_visited_controller.rb]: No, thanks 👍

File: app/controllers/phrase_usage_controller.rb (hunk 2 above)
Comment (sophiebrown01) [reply app/controllers/phrase_usage_controller.rb]: Also meant to delete, thanks

**Review by oscarwyatt — APPROVED** [review]
Noice

**Pull request merged**